### DISCUSSION TITLE
[Suggestion] Send notifications when someone stars your repo

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#47164] [Repositories]
Most other platforms notify you if someone likes your content, so Github should do the same

**Comment by OwaisAzad** [comment]
While it is true that many other platforms provide notifications for likes, I agree with you, but GitHub is primarily a platform for code collaboration and version control, rather than a social media platform. The focus of GitHub is on the collaboration and sharing of code, rather than receiving notifications for likes or other social interactions.

**Reply by adamlui** [reply]
This is true, perhaps an optional setting would be better than nothing

**Reply by adamlui** [reply]
Also an optional notification would increase user activity among those interested in this feature since we will click and end up browsing. It will even subsequently foster more engagement as users see it is typical behavior to star and follow, thus increasing the visibility of projects sitewide

**Comment by Grezor** [comment]
I think this feature is very interesting, it would let us know that other people are interested in our project. After on the other hand, it can be gimmicky, and on the other hand a spam notification.

I would put the idea of adding a small form when starring a project with questions: 
- Why did you star the project in order to improve the project 
- What do you like in the code (clean code, features ...) 
- Do you have recommendations to make ...

**Reply by adamlui** [reply]
I quite like your idea, sort of like reviewing apps which helps others decide whether to try it out themselves

**Reply by Grezor** [reply]
Yes, why not have a feedback from other developers to have more visibility on the project. It would also allow to have motivation :) After we have to see if they want to put this feature in practice.

**Reply by adamlui** [reply]
Why don't you post a suggestion for this awesome idea?

**Reply by goran-revops** [reply]
I believe adding a small form to capture this feedback would be a fantastic addition to the project. Good luck with the suggestion!

**Reply by Grezor** [reply]
Thanks for your feedback! I will search on the internet for some examples to put some captures

**Reply by Grezor** [reply]
Hello @adamlui, @rangola 

I have created a small satisfaction form. There might be some things to keep or delete. Feel free to give me feedback about my form. Try to create your own form, with different designs in order to have several ideas and visuals.

Thanks in advance for your feedback :)
ps : if you want to make a visual quickly : [excalidraw](https://excalidraw.com/)

![feedback_github](https://user-images.githubusercontent.com/38507456/218686648-70c73e92-a69b-4b45-9212-4b37d1875e4d.png)

**Reply by adamlui** [reply]
Yes that is good create the post and solicit opinions there so the engagement can help it rise to the top of suggestions

**Reply by goran-revops** [reply]
Yep I agree, looks good

**Comment by anandijain** [comment]
I would really like to be able to enable getting notified when someone stars my repo! 

It would help inform me who is looking at my code.

**Reply by Grezor** [reply]
Thank you for your message. Indeed it would allow to have a little more visibility for the less gifted to have a little recognition and visibility. Even to have opinions, or help :)
